### ISSUE TITLE
Don't show first bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [arkayenro/arkinventory#501]
```
Downloaded from > googlecode | wowinterface | curse | wowui | other (specify)

Curse

Awhile ago, I found a setting somewhere that let me hide the first bar (System 
Default). I always keep this bar empty, and since I can't size it, it just 
throws everything off. 

However, I made the mistake of resetting Arkinventory to fix another issue, and 
now I can't find this setting again. Where is it hiding?

Basically, my main problem is that the System Default bar has the preset, 
always showing slots in it. I don't want to see them. I have everything set not 
to show empty slots. 
```

Original issue reported on code.google.com by `Technica...@gmail.com` on 10 Feb 2012 at 8:29
